docs/content/users/providers/pantheon.md

```diff
@@ -31,8 +31,10 @@ If you have ddev installed, and have an active Pantheon account with an active s
 
 8. In your **project's** .ddev/providers directory, copy pantheon.yaml.example to pantheon.yaml (_Note that this refers to your project .ddev folder and not the global .ddev folder_).  Edit the `project` environment variable under `environment_variables`. It will be in the format `<projectname>.<environment>`, for example `yourprojectname.dev` or (in cases of ambiguity) `<project_uuid>.<environment>`, for example `009a2cda-2c22-4eee-8f9d-96f017321555.dev`.  
 
-9. `ddev restart`
+9. If using Colima, may need to set an explicit nameserver in `~/.colima/default/colima.yaml` like `1.1.1.1`. If this configuration is changed, may also need to [restart colima](https://ddev.readthedocs.io/en/stable/users/install/docker-installation/).
```

Comment:
@rfay - I notice the "restart Colima" link does not actually link directly to the macOS: Colima tab.  See: https://ddev--4076.org.readthedocs.build/en/4076/users/providers/pantheon/

Any suggestions here?

Reply:
Strangely, I see this works: https://ddev--4076.org.readthedocs.build/en/4076/users/install/docker-installation/#colima
But, this: https://ddev.readthedocs.io/en/stable/users/install/docker-installation/#colima

Does not?

Reply:
It should be a relative link in the docs, not a link to ddev.readthedocs.io, thanks for catching that.

Reply:
OTOH, I'm not sure a link is needed at all. But if a link, it should be a relative link.

Reply:
ok, I'll just remove the link